### PR TITLE
chore: add calcite-ui-icons to monorepo

### PR DESCRIPTION
**Pull request by benelan**
**Related Issue:** #9255

## Summary

Good news, everyone! `@esri/calcite-ui-icons` is joining the Calcite Design System monorepo. This change will ensure Calcite components always uses the latest and greatest version of the icons. Additionally, the icons will benefit from the monorepo's continuous integration, including automatic changelog generation and deployment. This will give the designers more time to do what they do best!

Some much needed cleanup was accomplished during this transition. The following items will no longer be [published to npm](https://unpkg.com/browse/@esri/calcite-ui-icons@3.29.0/):

```
.github/*
CONTRIBUTE.md
bin/build.js
bin/cli.js
bin/convert-mobile.js
bin/optimize.js
bin/path-data.js
bin/server.js
bin/templates/*
bower.json
docs/app.js
docs/index.html
docs/resources/*
launch-calcite.command
launch-meridian.command
```

Additionally, the autogenerated, build files will not longer be committed into version control. This includes:

```
docs/icons.json
index.d.ts
index.js
js/*
sprite-16.svg
sprite-24.svg
sprite-32.svg
```

These files can be generated via `npm run build`. The build files will be attached to GitHub releases for users who do not consume the icons via NPM.

Note that I removed `bower.json` since [they recommended](https://bower.io/blog/2017/how-to-migrate-away-from-bower/) using other software back in 2017, and there hasn't been any activity in the [repo](https://github.com/bower/bower) for a few years now. However, I'm open to deprecating bower support and removing it in the next major release if people prefer that route.

Release-As: 3.30.0

### CODE REVIEW

File: packages/calcite-ui-icons/README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# calcite-ui-icons
+
+A collection of UI SVG icons created by Esri for mapping applications.
+<https://esri.github.io/calcite-ui-icons/>
```
Comment (jcfranco) [inline packages/calcite-ui-icons/README.md:4]: Possibly out of scope, but should this point to the doc site?

File: packages/calcite-ui-icons/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# License
```
Comment (jcfranco) [inline packages/calcite-ui-icons/LICENSE.md:1]: Sidebar: we should ensure all packages have a license and licensing info included.

**Review by jcfranco — APPROVED** [review]
✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️
✴️⚛️⚛️⚛️⚛️✴️⚛️✴️✴️✴️✴️✴️⚛️✴️⚛️⚛️⚛️⚛️✴️✴️⚛️⚛️⚛️✴️✴️⚛️⚛️✴️✴️⚛️✴️✴️✴️⚛️✴️⚛️⚛️⚛️⚛️✴️⚛️✴️
✴️⚛️✴️✴️⚛️✴️⚛️✴️✴️✴️✴️✴️⚛️✴️⚛️✴️✴️✴️✴️⚛️✴️✴️✴️✴️⚛️✴️✴️⚛️✴️⚛️⚛️✴️⚛️⚛️✴️⚛️✴️✴️✴️✴️⚛️✴️
✴️⚛️⚛️⚛️⚛️✴️⚛️✴️✴️⚛️✴️✴️⚛️✴️⚛️⚛️⚛️✴️✴️✴️⚛️⚛️✴️✴️⚛️✴️✴️⚛️✴️⚛️✴️⚛️✴️⚛️✴️⚛️⚛️⚛️✴️✴️⚛️✴️
✴️⚛️✴️✴️⚛️✴️⚛️✴️⚛️✴️⚛️✴️⚛️✴️⚛️✴️✴️✴️✴️✴️✴️✴️⚛️✴️⚛️✴️✴️⚛️✴️⚛️✴️✴️✴️⚛️✴️⚛️✴️✴️✴️✴️✴️✴️
✴️⚛️✴️✴️⚛️✴️✴️⚛️✴️✴️✴️⚛️✴️✴️⚛️⚛️⚛️⚛️✴️⚛️⚛️⚛️✴️✴️✴️⚛️⚛️✴️✴️⚛️✴️✴️✴️⚛️✴️⚛️⚛️⚛️⚛️✴️⚛️✴️
✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️✴️

File: packages/calcite-ui-icons/LICENSE.md (hunk 2 above)
Comment (benelan) [reply packages/calcite-ui-icons/LICENSE.md:1]: I remembered to do this when creating the monorepo! It looks like the tokens don't have the `license` field in package.json though, I'll add that in a separate PR.

**Pull request merged**